armi/physics/neutronics/globalFlux/globalFluxInterface.py

```diff
@@ -327,6 +328,7 @@ def fromUserSettings(self, cs):
         self.adjoint = neutronics.adjointCalculationRequested(cs)
         self.real = neutronics.realCalculationRequested(cs)
         self.detailedAxialExpansion = cs["detailedAxialExpansion"]
+        self.nonUniformAssems = len(cs["nonUniformAssemFlags"]) > 0
```

Comment:
Can this just be `self.nonUniformAssems = cs["nonUniformAssemFlags"]`?

Reply:
`self.nonUniformAssems` is supposed to be a bool, but `cs["nonUniformAssemFlags"]` is a list.
Technically, it `self.nonUniformAssems` is a list it would probably evaluate to the correct `bool` value when we use it in logical statements, but it seems better for it to explicitly be a bool.

Reply:
Minor comment, but consider making this `hasNonUniformAssems`.

Reply:
Let's do any([]) then